common/scripts/update-npm-dependencies.js

```diff
@@ -0,0 +1,141 @@
+const ncu = require('npm-check-updates');
+const Octokit = require('@octokit/rest');
+const { installAndRun, findRushJsonFolder, RUSH_JSON_FILENAME } = require("./install-run");
+const fs = require('fs');
+const { join } = require('path');
+const { promisify } = require('util');
+const { spawnSync } = require('child_process');
+const readdir = promisify(fs.readdir);
```

Comment:
Can you use `readdirSync` instead? This code is intended to be run as a script and the sync versions are really best suited for scripts.

Reply:
changed